common/script/statHelpers.js

```diff
@@ -0,0 +1,55 @@
+/*
+  ------------------------------------------------------
+  Level cap
+  ------------------------------------------------------
+ */
+
+const maxLevel = 100;
+
+function capByLevel (lvl) {
+  if (lvl > maxLevel) {
+    return maxLevel;
+  } else {
+    return lvl;
+  }
+}
+
+/*
+  ------------------------------------------------------
+  Health cap
+  ------------------------------------------------------
+ */
+
+const maxHealth = 50;
+
+/*
+  ------------------------------------------------------
+  Scoring
+  ------------------------------------------------------
+ */
+
+function tnl (lvl) {
```

Comment:
Let's give this a full name: `toNextLevel`

We can keep it as `api.tnl` in the main file.